test/AuthenticationRequest.request.spec.ts

```diff
@@ -78,6 +78,7 @@ describe("create Request Uri should", () => {
             registration: {
                 didMethodsSupported: ['did:ethr:'],
                 subjectIdentifiersSupported: SubjectIdentifierType.DID,
+                credential_formats_supported: CredentialType.JWT,
```

Comment:
TBH i think PE support for JWT based VCs is not complete, so it might make more sense to start with json_ld type based credentials in our tests as well

Reply:
Fixed!